tools/ci/check_grep.sh

```diff
@@ -54,6 +54,11 @@ if grep -Pzo '"\w+" = \(\n[^)]*?/obj/machinery/atmospherics/pipe/(?<type>[/\w]*)
     echo "ERROR: found multiple identical pipes on the same tile, please remove them."
     st=1
 fi;
+if grep -rzoP 'machinery/door.*{([^}]|\n)*name = .*("|\s)(?!of|and|to)[a-z].*\n' --include=*.dmm;	then
```

Comment:
```suggestion
if grep -rzoP 'machinery/door.*{([^}]|\n)*name = .*("|\s)(?!of|and|to)[a-z].*\n' _/maps/**/*.dmm;	then
```

Have it mirror the rest of the greps? This probably works but I just want to make doubly extra sure that something somewhere doesn't break. I know you said you didn't have Windows Subsystem for Linux the other day so if you think it's fine commit this suggestion and I'll test it out on my machine.

Reply:
The `--include` flag is more portable (doesn't require use of the shopt globstar builtin) and works fine on Ubuntu 20.04. However that's a battle for another day, san is right - better to fit with the other greps for now.

Still, do not commit this, as there's a minor typo - the path should read `_maps/**/*.dmm`